.gitignore

```diff
@@ -22,3 +22,6 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 cx_leaderboard-*.tar
 
+\.idea
```

Comment:
Right now we don't have any editor-specific ignores in .gitignore. Do you think moving these into your personal home directory's global git ignore would be sufficient? That way you wouldn't need to ignore these files on individual projects.